content/docs/command-reference/import-url.md

```diff
@@ -37,25 +36,27 @@ later, if/when the external data source changes. Example scenarios:
 
 `dvc import-url` helps you create such an external data dependency, without
 having to manually copy files from the supported locations (listed below), which
-may require installing a different tool for each type.
-
-When you don't want to store the target data in your local system, you can still
-create an import `.dvc` file while transferring a file or directory directly to
-remote storage, by using the `--to-remote` option. See the
-[Transfer to remote storage](#example-transfer-to-remote-storage) example for
-more details.
+would require installing/using a different tool for each type.
 
 The `url` argument specifies the external location of the data to be imported.
 The imported data is <abbr>cached</abbr>, and linked (or copied) to the current
-working directory with its original file name e.g. `data.txt` (or to a location
-provided with `out`).
+working directory with its original file name e.g. `data.txt`, or to a location
+provided with `out`.
 
 An _import `.dvc` file_ is created in the same location e.g. `data.txt.dvc` –
-similar to using `dvc add` after downloading the data. This makes it possible to
-update the import later, if the data source has changed (see `dvc update`).
+similar to using `dvc add` after downloading the data. It saves the information
+about the data source, so the import can be updated later if the data source has
+changed (see `dvc update`).
+
+💡 Using an
+[external cache](/doc/use-cases/shared-development-server#configure-the-external-shared-cache)
+or the `--to-remote` option lets you
+[transfer](#example-transfer-to-remote-storage) an import without using the
+local file system.
```

Comment:
Note: I realized the to-cache strategy also applies to regular `import` (I think), so I added this note.